pkg/predictor_source/inferenceservice_registry.go

```diff
@@ -39,6 +40,7 @@ const (
 	runtimeAnnotation    = "serving.kserve.io/servingRuntime"
 
 	azureBlobHostSuffix = "blob.core.windows.net"
+	pvcStorageType      = "pvc"
```

Comment:
I don't think a constant is needed here, best to keep it consistent with the others.